nengo_gui/static/lib/js/interact-1.2.4.js

```diff
@@ -3354,6 +3354,7 @@
         this.shiftKey  = event.shiftKey;
         this.metaKey   = event.metaKey;
         this.button    = event.button;
+        this.buttons   = event.buttons;
```

Comment:
Also, [it seems that this is fixed in more recent versions of interact.js](https://github.com/taye/interact.js/commit/d384fcc47cbb0f2f47c65dd09e7855befd646522). I would prefer to update interact.js instead of rolling our own patch.